packages/menu-bar/src/vaadin-menu-bar-submenu.js

```diff
@@ -21,7 +22,7 @@ import { SubMenuMixin } from './vaadin-menu-bar-submenu-mixin.js';
  * @mixes ThemePropertyMixin
  * @protected
  */
-class MenuBarSubmenu extends SubMenuMixin(ThemePropertyMixin(PolylitMixin(LitElement))) {
+class MenuBarSubmenu extends SubMenuMixin(ThemePropertyMixin(CSSInjectionMixin(PolylitMixin(LitElement)))) {
```

Comment:
The submenu isn't supposed to be stylable so the mixin can be removed from here.